src/s3ql/mount.py

```diff
@@ -529,6 +533,8 @@ def compression_type(s):
                            'user and the root user.')
     parser.add_argument("--fg", action="store_true", default=False,
                       help="Do not daemonize, stay in foreground")
+    parser.add_argument("--fs-name", default='',
+                      help="Mount name passed to fuse.")
```

Comment:
Can we be more specific here? Eg "Will be shown by mount in the <foo> column" (I can't actually check where this goes right now)

Reply:
Sure, that is where it goes btw.